### PR TITLE
Add new webinars

### LINKED ISSUE #1905: Add new webinars
We want to organise some more webinars. So we need new landing pages where people can subscribe, like this one https://www.defactolearning.de/webinar

They should also be included on the events page https://www.defactolearning.de/veranstaltungskalender/

Webinar dates are:
04.07. 10:00 Uhr Curated Learning: Zielorientiertes Lernen managen
16.07. 10:00 Uhr Compliance und Qualifikationsmanagement
18.07. 10:00 Uhr Compliance und Qualifikationsmanagement
03.09. 10:00 Uhr Performance Support

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
Close #1905

### CODE REVIEW

**Comment by RonaldDijkstra** [comment]
We now have this: https://www.defactolearning.de/webinar 

In this PR I redirect that one to the first one mentioned in the issue. 
The others I added and I added them to the veranstaltungen. 

The CircleCI failed tests are the missing pages which need to be deployed first: 

```
- ./build/webinar-compliance-und-qualifikationsmanagement-16-07-2019/index.html
  *  External link https://www.defactolearning.de/webinar-compliance-und-qualifikationsmanagement-16-07-2019/ failed: 404 No error
- ./build/webinar-compliance-und-qualifikationsmanagement-18-07-2019/index.html
  *  External link https://www.defactolearning.de/webinar-compliance-und-qualifikationsmanagement-18-07-2019/ failed: 404 No error
- ./build/webinar-curated-learning-04-07-2019/index.html
  *  External link https://www.defactolearning.de/webinar-curated-learning-04-07-2019/ failed: 404 No error
- ./build/webinar-performance-support-03-09-2019/index.html
  *  External link https://www.defactolearning.de/webinar-performance-support-03-09-2019/ failed: 404 No error
```
@isabelamy you can review it with this deploy preview link: 

https://deploy-preview-1927--defacto-de.netlify.com/veranstaltungskalender/

**Comment by RonaldDijkstra** [comment]
All webinars have the same endpoint in Formkeep.

**Comment by RonaldDijkstra** [comment]
Redirect test: https://deploy-preview-1927--defacto-de.netlify.com/webinar

File: source/localizable/evenementen/_events.de.md.erb [hunk 1]
```diff
@@ -1,59 +1,168 @@
...
+    <div class="col right">
       <h2>
-        Zukunft Personal
+        Webinar: 'Performance Support'
       </h2>
       <p class="meta">
-        <span>17. - 18. September 2019</span>
-        <span class="divider">|</span>
-        <span>Köln</span>
+        <span>18. Juli 2019</span>
```
Comment (isabelamy) [inline source/localizable/evenementen/_events.de.md.erb]: 03. September 2019

**Pull request merged**